src/DynamoCore/DSEngine/CodeCompletionServices.cs

```diff
@@ -19,10 +20,8 @@ public class CodeCompletionServices
         public static string[] KeywordList = {Keyword.Def, 
                                         Keyword.If, Keyword.Elseif, Keyword.Else, 
                                         Keyword.While, Keyword.For, Keyword.In, Keyword.Continue,  
-                                        Keyword.Int, Keyword.Double, Keyword.String, Keyword.Bool, Keyword.Char, 
```

Comment:
I removed these as these are primitive types that can also be obtained from the `classtable`. Earlier I was recovering only user defined types from the `classtable`. Now I'm recovering these as well because of which I don't have to add these to the keyword list separately.